engine/src/test/scala/cromwell/services/KeyValueServiceActorSpec.scala

```diff
@@ -0,0 +1,186 @@
+package cromwell.services
+
+import akka.pattern.ask
+import akka.util.Timeout
+import com.typesafe.config.ConfigFactory
+import cromwell.CromwellTestkitSpec.TestWorkflowManagerSystem
+import cromwell.engine.WorkflowSourceFiles
+import cromwell.engine.backend.local.LocalBackend
+import cromwell.engine.db.slick.SlickDataAccess
+import cromwell.services.KeyValueServiceActor._
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class KeyValueServiceActorSpec extends FlatSpec with Matchers with CromwellServicesTest {
+  val workflowManagerSystem = new TestWorkflowManagerSystem
+  val actorSystem = workflowManagerSystem.actorSystem
+  val dataAccess = new SlickDataAccess(ConfigFactory.parseString(
```

Comment:
Are you testing some specific variation of data access? Otherwise, convention should be to use the "main" data access code, that for now is the singleton.